### PR TITLE
Update run w/parameters icon and increment parameterized notebook

### PR DESCRIPTION
**Pull request by VasuBhog**
This PR updates the icon and focuses on incrementing the parameterized notebook based on the original notebook title (shown below).

https://user-images.githubusercontent.com/23587151/114684373-b428df80-9cc5-11eb-87b9-f955c4ba56fd.mp4

New Icon:
image

### CODE REVIEW

**Review by lucyzhang929 — APPROVED** [review]
There's a failing unit test.

File: src/sql/workbench/contrib/notebook/browser/notebookActions.ts [hunk 1]
```diff
@@ -364,15 +364,39 @@ export class RunParametersAction extends TooltipFromLabelAction {
...
 	}
 }
 
+/**
+ * Will iterate the title of the parameterized notebook since the original notebook is still open
+ * @param originalTitle is the title of the original notebook that we run parameterized action from
+ * @param listNotebookEditors is the list of notebooks editors currently open
+ * @returns the title of the parameterized notebook
+ */
+export function getUntitledUriPath(originalTitle: string, listNotebookEditors: INotebookEditor[]): string {
```
Comment (Charles-Gagnon) [inline src/sql/workbench/contrib/notebook/browser/notebookActions.ts:383]: This seems like a more generally useful thing - not really specific to parameterized notebooks (we already do something similar in the extension).

I'd suggest making this a common function in the service code so it's more easily discoverable

**Review by Charles-Gagnon — APPROVED** [review]
Change looks fine but there's the failing test - and I would like to see a few simple tests for the new function being added.

**Review by barbaravaldez — APPROVED** [review]

**Comment by VasuBhog** [comment]
> Change looks fine but there's the failing test - and I would like to see a few simple tests for the new function is added.

Added a simple test. But saw that the notebookService suite is being skipped. Will be working on getting those tests unskipped in another PR in order to separate the work properly.

**Review by MaddyDev — APPROVED** [review]

**Pull request merged**